scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 547,
-  MAX_SIZE: 585,
+  MAX_SIZE: 591,
```

Comment:
Doesn't increase the bundles this much - Persian just tipped over 586k, so reset the upper limit to +5kb

Reply:
does the lower limit need increasing as well?

Reply:
Yep - smallest bundle is sport (?!) at 555k